telemetry_core/src/main/java/com/newrelic/telemetry/Attributes.java

```diff
@@ -25,6 +25,16 @@
 public class Attributes {
   private final Map<String, Object> rawAttributes = new HashMap<>();
 
+  public Attributes() {}
+
+  public Attributes(Attributes other) {
```

Comment:
I think we might want this constructor to only be private. Let's verify.

Reply:
Thoughts on "original" instead of "other?

Reply:
we could just call it `attributes`

Reply:
I'm okay with "original".